pkgs/development/libraries/qmltermwidget/default.nix

```diff
@@ -28,7 +48,12 @@ stdenv.mkDerivation {
       --replace '$$[QT_INSTALL_QML]' "/$qtQmlPrefix/"
   '';
 
-  installFlags = [ "INSTALL_ROOT=$(out)" ];
+  installFlags = [ "INSTALL_ROOT=${placeholder "out"}" ];
+
+  preInstall = ''
+    mkdir -p $out/lib/qt-5.15.9/qml//QMLTermWidget/kb-layouts/
+    touch $out/lib/qt-5.15.9/qml//QMLTermWidget/kb-layouts/macbook.keytab
```

Comment:
The version number must be substituted

Reply:
This was just an experiment to debug the install phase.